src/sentry/tasks/deletion.py

```diff
@@ -267,6 +267,48 @@ def delete_group(object_id, transaction_id=None, **kwargs):
         )
 
 
+@instrumented_task(
+    name='sentry.tasks.deletion.delete_groups',
+    queue='cleanup',
+    default_retry_delay=60 * 5,
+    max_retries=MAX_RETRIES
+)
+@retry(exclude=(DeleteAborted, ))
+def delete_groups(object_ids, transaction_id=None, eventstream_state=None, **kwargs):
+    from sentry import deletions, eventstream
+    from sentry.models import Group
+
+    if not object_ids:
+        return
+
+    object_ids = sorted(object_ids)
```

Comment:
I think it makes sense that this is ordered. I think you could make a reasonable argument for either `times_seen` or `last_seen`… it really just depends on how you use the product, I guess? It might make sense to have these already sorted as arguments rather than sorting them here.